css/css-ui/outline-017.html

```diff
@@ -29,20 +29,24 @@
   <div id=log></div>
 
   <script>
-    test(
-      function(){
-        var test = document.getElementById("test");
-        assert_equals(getComputedStyle(test).outlineColor, 'rgb(2, 0, 0)');
-      }, "outline-color is animated as a color");
-    test(
-      function(){
-        var test = document.getElementById("test");
-        assert_equals(getComputedStyle(test).outlineWidth, '2px');
-      }, "outline-width is animated as a length");
-    test(
-      function(){
-        var test = document.getElementById("test");
-        assert_equals(getComputedStyle(test).outlineOffset, '2px');
-      }, "outline-offset is animated as a length");
+    setup({explicit_done:true});
```

Comment:
It would seem better to use `async_test` for this rather than `test` with `explicit_done`.